qemu/tests/win_virtio_update.py

```diff
@@ -193,21 +207,32 @@ def check_cdrom(timeout):
         if install_cmds:
             cmd = re.sub("WIN_UTILS", vol_utils, install_cmds[driver])
             cmd = re.sub("WIN_VIRTIO", vol_virtio, cmd)
-            session.cmd(cmd, timeout=driver_install_timeout)
-            session = vm.reboot(session)
+            f_flag_tmp, log_tmp = virtio_driver_install(session, cmd, driver)
+            if f_flag_tmp:
+                fail_flag = True
+                fail_log += log_tmp
+            session = vm.reboot(nic_index=nic_index)
+    driver_debug_file = params_driver.get("driver_debug_file").split(',')
+    error.context("Get Driver installation log", logging.info)
+    for _file in driver_debug_file:
+        s, o = session.cmd_status_output("dir %s" % _file)
+        if not s:
+            output = session.cmd("type %s" % _file)
+            _file_host = _file.split("\\")[-1].strip()
```

Comment:
Normally we don't name a variable start with _:

Please check this:

https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles